dspace-api/src/main/java/org/dspace/content/DSpaceObject.java

```diff
@@ -76,6 +76,12 @@
     @Transient
     private boolean modified = false;
 
+    @Transient
+    protected UUID predefinedUUID;
+    public UUID getPredefinedUUID() {
```

Comment:
This should be described in JavaDocs/comments... as it's not entirely clear when looking at DSpaceObject what this property *does*.  We should describe that this is read by our UUID generator to assign a predefined UUID (if specified).